themes/bubble-wrap-icons.yaml

```diff
@@ -3,7 +3,7 @@ global:
     # are otherwise defined in the root {style}-style.yaml,
     # themes/color-{theme}.yaml or themes/label-{step} files.
     # This also means {style}-icons.yaml should be imported first.
-    icon_size:          [[13, '14px'], [16, '18px'], [18, '19px']]
+    icon_size:          [[7,[75%,75%]],[12,[100%,100%]],[15,[110%,110%]]]
```

Comment:
Can be `icon_size: [[7, 75%], [12, 100%], [15, 110%]]`

Reply:
ce05472e6916d311fcef4071e6b1f7d53d629295

Reply:
and d36fedbfef2b5fdf8dadb801f59c8d4bcdfc91ac